### PR TITLE
changed the overview link for the markdown file to show the pdf as em…

### LINKED ISSUE #2242: Update how 'Overview' link opens for LA TDM Calculator
### Overview
As a user, clicking on the 'Overview' link for the LA TDM Calculator project should open a new tab to the pdf file directly. This is made as an action item of #2211 

### Details
When clicking the 'Overview' link, each project page acts differently. Some open to a google docs page, others open to a github page that hosts the pdf file, and another downloads the pdf file locally to the machine. There needs to be a standard in how the pdf files open. Developers decided during the backend meeting that pdfs should be opened as a separate tab to the file directly, since modern browsers have the ability to view pdfs.

**Example:**
[How LA TDM Calculator's pdf file currently opens](https://github.com/hackforla/product-management/blob/master/project-one-sheets/TDM-Calculator-Product-One-Sheet.pdf)
[How it should open](https://www.hackforla.org/test/Access-the-Data-One-Sheet.pdf)

### Action Items
- [ ] Download a copy of LA TDM Calculator's overview pdf and put it in [/assets/pdfs](https://github.com/hackforla/website/tree/gh-pages/assets/pdfs)
- [ ] Change the 'Overview' link for the [markdown file](https://github.com/hackforla/website/blob/gh-pages/_projects/tdm-calculator.md) to point to the pdf file living under /assets/pdfs
- [ ] Ensure that the links in both the [project page](https://www.hackforla.org/projects/) and the project's [dedicated page](https://www.hackforla.org/projects/tdm-calculator) open the pdf file properly after the change

### Resources/Instructions
[Markdown file](https://github.com/hackforla/website/blob/gh-pages/_projects/tdm-calculator.md)
[Project Page](https://www.hackforla.org/projects/tdm-calculator)
[Location for pdf files](https://github.com/hackforla/website/tree/gh-pages/assets/pdfs)
[Jekyll Static Files](https://jekyllrb.com/docs/static-files/)

### PR DESCRIPTION
**Pull request by blulady**
…bedded

Fixes #2242

### What changes did you make and why did you make them ?

  -I added the pdf to the assets/pdf
  -I added the link to the TDM_Calculater.md 

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)

<details>
<summary>Visuals before changes are applied</summary>

image

</details>

<details>
<summary>Visuals after changes are applied</summary>

image

### CODE REVIEW

**Comment by JessicaLucindaCheng** [comment]
@blulady In your original post, where it says "Fixes #update-how-overview-link-opens-for-LA-TDM-Calculator_2242", can you edit it so it links to the issue this pr is supposed to fix? So, it should be `Fixes #2242`.

**Comment by blulady** [comment]
Thanks Jessica!! I knew I had done something.

**Comment by JessicaLucindaCheng** [comment]
@blulady 
On the right, please add your pull request to the project board.
Add-to-project-board

**Comment by JessicaLucindaCheng** [comment]
- Availability: 1 hour
- Review ETA: Sun, Feb 13, 2022

**Review by JessicaLucindaCheng — CHANGES_REQUESTED** [review]
@blulady Thanks for working on this issue!
- [ ] On the right, please add your pull request to the project board.
Add-to-project-board

- [ ] I don't see LA TDM Calculator's overview pdf in the /assets/pdfs directory. So, please add the pdf there.
- [ ] The link should not be set in embed tags. The link should be changed and should look something like `'../assets/pdfs/Name-of-One-Sheet.pdf'` with Name-of-One-Sheet,pdf replaced with the actual name of the pdf file you added to the /assets/pdfs folder.

Please take a look at the instructions in your issue #2242 again for more details.

**Comment by blulady** [comment]
Ok... So I don't know how I got so far off track. But I believe I made the correct fixes this time. Can I submit an entirely new pr for this issue?

**Comment by blulady** [comment]
Also can you send a page link to where I add the pr to the project board?

**Comment by JessicaLucindaCheng** [comment]
@blulady 

> Ok... So I don't know how I got so far off track. But I believe I made the correct fixes this time. Can I submit an entirely new pr for this issue?

You don't need to open a new pr. If you are using the same topic branch (update-how-overview-link-opens-for-LA-TDM-Calculator_2242) in your forked repo, you can just push commits to your topic branch and you should see the commits in this pr.


> Also can you send a page link to where I add the pr to the project board?

Sorry, I wasn't clearer about how to add your pr to the Project Board. Hopefully, the following instructions are more helpful:

1) On the right, under the "Projects" section
how-to-add-to-the-project-board1

2) Click on the gear icon
Add-to-project-board

3) Then click on Project Board to add your pull request to the project board.
how-to-add-to-the-project-board3

**Comment by JessicaLucindaCheng** [comment]
@blulady I have removed the request for review for myself for now. Please make the requested changes in this comment: https://github.com/hackforla/website/pull/2772#pullrequestreview-881115088

After you make those changes and you are ready for me to re-review your pull request, please click the following in the Reviewers section on the right:
image

**Comment by blulady** [comment]
I think I corrected it ¯\_(ツ)_/¯ 
Let me know

**Comment by JessicaLucindaCheng** [comment]
- Availability: 1 hour
- Re-review ETA: Tue, Feb 22, 2022

File: _projects/tdm-calculator.md [hunk 1]
```diff
@@ -51,7 +51,11 @@ links:
   - name: Slack
     url: 'https://hackforla.slack.com/archives/CKY65G266'
   - name: Overview
-    url: https://github.com/hackforla/product-management/blob/master/project-one-sheets/TDM-Calculator-Product-One-Sheet.pdf
+<<<<<<< HEAD
+    url: ../assets/pdfs/TDM-Calculator-Product-One-Sheet.pdf
+=======
+    [embed]url: https://github.com/hackforla/product-management/blob/master/project-one-sheets/TDM-Calculator-Product-One-Sheet.pdf[/embed]
+>>>>>>> 5277539778f9c03ef1a6839d8feb01a4179324b5
```
Comment (JessicaLucindaCheng) [inline _projects/tdm-calculator.md:58]: Lines 56-58 arose from a merge conflict. These lines should be removed.

Comment (JessicaLucindaCheng) [inline _projects/tdm-calculator.md]: Lines 56-58 arose from a merge conflict. These lines should be removed.

File: _projects/tdm-calculator.md [hunk 2]
```diff
@@ -51,7 +51,11 @@ links:
   - name: Slack
     url: 'https://hackforla.slack.com/archives/CKY65G266'
   - name: Overview
-    url: https://github.com/hackforla/product-management/blob/master/project-one-sheets/TDM-Calculator-Product-One-Sheet.pdf
+<<<<<<< HEAD
```
Comment (JessicaLucindaCheng) [inline _projects/tdm-calculator.md]: Line 54 arose from a merge conflict. This line should be removed.

**Review by JessicaLucindaCheng — CHANGES_REQUESTED** [review]
@blulady Thanks for adding the pdf to assets/pdfs and making the requested changes!

- [ ] However, since code from different branches were merged together and the same file was changed, there is now a merge conflict in the file `tdm-calculator.md`. You will need to edit the file and push changes to fix this issue. (See my in-line comments for which lines should be removed.)

- [ ] When you are ready for me to re-review your pull request, please click the following in the Reviewers section on the right:
image

**Review by JessicaLucindaCheng — APPROVED** [review]
@blulady Good job with the changes! You did it!

**Pull request merged**